Rakefile

```diff
@@ -25,6 +25,18 @@ task :test_swift do
   end
 end
 
+desc "Generates Swift source code and run its unit tests with no infer types."
+task :test_swift_with_no_infer_types do
```

Comment:
Could you move the logic inside this task as well as the Kotlin one into the original `test_swift` and `test_kotlin` tasks, so that it merges them together? Like this pseudocode:

```ruby
task :test_swift do
  # Run the test that we were already doing before
  # Run this new test here where infer_types=false.
  # Make sure to reuse whatever code is possible between the two steps above ☝️ , if any
end
```